From 441fc4d8017a1b2e53a2303fe4afa5cc85b15c2f Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Fri, 26 Jul 2013 14:36:29 +0800
Subject: [PATCH 34/39] ENGR00272735 Remove duplicate
 RIL_REQUEST_BASEBAND_VERSION handling

RIL_REQUEST_BASEBAND_VERSION already been handled in jb4.3 code

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 reference-ril/reference-ril.c | 11 ++++-------
 1 file changed, 4 insertions(+), 7 deletions(-)

diff --git a/reference-ril/reference-ril.c b/reference-ril/reference-ril.c
index 7830e1b..9840535 100755
--- a/reference-ril/reference-ril.c
+++ b/reference-ril/reference-ril.c
@@ -3344,6 +3344,10 @@ onRequest (int request, void *data, size_t datalen, RIL_Token t)
                 requestBasebandVersion(data, datalen, t);
                 break;
             }
+            else {
+                RIL_onRequestComplete(t, RIL_E_REQUEST_NOT_SUPPORTED, NULL, 0);
+                break;
+            }
 
         case RIL_REQUEST_DEVICE_IDENTITY:
             if (TECH_BIT(sMdmInfo) == MDM_CDMA) {
@@ -3415,13 +3419,6 @@ onRequest (int request, void *data, size_t datalen, RIL_Token t)
             requestSetCellInfoListRate(data, datalen, t);
             break;
 
-        case RIL_REQUEST_BASEBAND_VERSION:
-            if (current_modem_type == HUAWEI_MODEM)
-                requestBasebandVersion(data, datalen, t);
-            else
-                RIL_onRequestComplete(t, RIL_E_REQUEST_NOT_SUPPORTED, NULL, 0);
-            break;
-
         default:
             RLOGD("Request not supported. Tech: %d",TECH(sMdmInfo));
             RIL_onRequestComplete(t, RIL_E_REQUEST_NOT_SUPPORTED, NULL, 0);
-- 
1.8.0

